src/alfanous-django/templates/base.html

```diff
@@ -372,8 +372,9 @@ <h1>
                             {% endif %}
                         </form>
                 </div><!--/span4-->
-            </main>
+
             </div><!--/row-->
+            </main>
```

Comment:
This way the results are not contained in the main block. 

To include results block, We have to include side nav too. only if we take out.